src/main/groovy/com/getkeepsafe/dexcount/DexMethodCountTask.groovy

```diff
@@ -66,6 +67,7 @@ class DexMethodCountTask extends DefaultTask {
             printFullTree()
             printChart()
             printTaskDiagnosticData()
+            failBuildMaxMethods()
```

Comment:
Run this method last, so the user(developer) can see the exception after the reports have printed to STDOUT and saved to disk.